code/modules/mob/living/simple_animal/abnormality/teth/forsaken_murderer.dm

```diff
@@ -0,0 +1,72 @@
+/mob/living/simple_animal/hostile/abnormality/forsaken_murderer //designed to be a very forgiving and standard abnormality.
+	name = "forsaken murderer"
+	desc = "A unhealthy looking human bound in a full body straightjacket. His neck is broken and in the middle of his forehead is a old wound that refuses to heal."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "forsakenmurdererinert"
+	icon_living = "forsakenmurdererinert"
+	icon_dead = "forsakenmurdererdead"
+	del_on_death = FALSE
+	maxHealth = 600 //originally was 270. Fragment health is 800 with a original game health of 230 so techically forsaken murderer has more health than fragment? Ill round the numbers to 600 since 270 can be rounded to 300 and doubled.
```

Comment:
Feel free to give it more health. Fragment's main difficulty comes from it having an AoE that BUFFS THE THING. If we do the funky math, 900 would be a good number. Especially considering it has very nice resistances